src/main/java/de/geolykt/enchantments_plus/CustomEnchantment.java

```diff
@@ -812,15 +812,21 @@ public PersistentDataGatherer() {
         @Override
         public LinkedHashMap<CustomEnchantment, Integer> getEnchants(ItemStack stk, boolean acceptBooks, World world,
                 List<String> outExtraLore) {
-            
+
             LinkedHashMap<CustomEnchantment, Integer> map = new LinkedHashMap<>();
             if ( (stk != null && stk.getType() != Material.AIR) && (acceptBooks || stk.getType() != Material.ENCHANTED_BOOK)) {
                 if (stk.hasItemMeta()) {
                     final PersistentDataContainer cont = stk.getItemMeta().getPersistentDataContainer();
-                    
+
                     if (doCompat && cont.getOrDefault(ench_converted, PersistentDataType.BYTE, (byte) 0) == 0) {
                         //Legacy conversion
-                        Bukkit.getLogger().info("Item converted");
+                        Bukkit.getLogger().info(
+                                String.format("[%s] Item converted (%s)", 
+                                    world.getName(), 
+                                    stk.getType().toString()
+                                    )
+                                );
+
```

Comment:
This is actually a debug statement I forgot to remove, it is not meant to stay within the code
```suggestion
```